source/docs/api-reference/index.rst

```diff
@@ -3,6 +3,29 @@
 API Reference
 =============
 
+This section highlights the functionality that is exposed as part of the Phoenix Pro API. This includes the base API, simulation and detailed example projects.
```

Comment:
Add Oxford comma: "base API, simulation, and detailed example projects"

Reply:
I've typically only used Oxford comma in lists of greater than 3, but I'll add one here anyways. Doesn't affect readability either way.